test/EFCore.MySql.FunctionalTests/appsettings.json

```diff
@@ -2,9 +2,9 @@
   "Logging": {
     "IncludeScopes": false,
     "LogLevel": {
-      "Default": "Error",
-      "System": "Error",
-      "Microsoft": "Error"
+      "Default": "Trace",
```

Comment:
Just push another commut to your branch that changes it back. I'll do a Squash Merge and it will be like it never happened 😄